utils/locale.js

```diff
@@ -1,9 +1,17 @@
 /* eslint-disable import/prefer-default-export */
 
+const defaultLocale = 'de'
+
 export function getLocaleFromQuery(queryObject) {
   if (queryObject && queryObject.locale) {
     return queryObject.locale
   }
+  return defaultLocale
+}
 
-  return 'de'
+export function formatDateForLocale(locale, date) {
+  const effectiveLocale = locale || defaultLocale
+  const options = { day: '2-digit', month: '2-digit', year: 'numeric' }
+  const format = new Intl.DateTimeFormat(effectiveLocale, options)
```

Comment:
I think @torkjels was trying this out, but then he changed his mind. I think the reason is that nodejs environments are different and sometimes they don't have the `Intl` which means you need to polyfill them. Otherwise, the statically rendered date will be different.

Reply:
It seems it can be switched on in node.
https://stackoverflow.com/questions/39626119/where-does-intl-numberformat-support-come-from-in-node-js

https://nodejs.org/api/intl.html#intl_internationalization_support

Reply:
By the way, we can change the node version in `.nvmrc` for netlify. At the moment is set to latest node version 8.